tests/auth/test_auth_route.py

```diff
@@ -87,7 +87,7 @@ def test_login_wrong_credentials(self):
             "email": self.person_dict["email"],
             "password": "wrongpassword",
         }
-        result = self.post("auth/login", credentials, 400)
```

Comment:
Wrong credentials should return a 400 error, it's a problem with the param not the access to the route.

Reply:
An RFC (7235) says that it should return 401 as status code. 
https://stackoverflow.com/questions/32752578/whats-the-appropriate-http-status-code-to-return-if-a-user-tries-logging-in-wit